packages/main/src/BusyIndicator.js

```diff
@@ -0,0 +1,101 @@
+import UI5Element from "@ui5/webcomponents-base/src/UI5Element.js";
+import Bootstrap from "@ui5/webcomponents-base/src/Bootstrap.js";
+import BusyIndicatorTemplateContext from "./BusyIndicatorTemplateContext.js";
+import BusyIndicatorRenderer from "./build/compiled/BusyIndicatorRenderer.lit.js";
+import Icon from "./Icon.js";
+
+// Styles
+import busyIndicatorCss from "./themes/BusyIndicator.css.js";
+
+// all themes should work via the convenience import (inlined now, switch to json when elements can be imported individyally)
+import "./ThemePropertiesProvider.js";
+import BusyIndicatorType from "./types/BusyIndicatorType.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-busy-indicator",
+	usesNodeText: true,
+	properties: /** @lends sap.ui.webcomponents.main.BusyIndicator.prototype */ {
+		/**
+		 * Defines the size of the <code>ui5-busy-indicator</code>.
+		 * </br></br>
+		 * <b>Note:</b> Available options are "Small", "Medium", "Large"
+		 *
+		 * @type {BusyIndicatorType}
+		 * @defaultvalue "Large"
+		 * @public
+		 */
+		size: { type: BusyIndicatorType, defaultValue: BusyIndicatorType.Large },
+		/**
+		 * Defines if the <code>ui5-busy-indicator</code> is visible.
+		 * </br></br>
+		 * <b>Note:</b> Available options are true and false
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		invisible: { type: Boolean, defaultValue: false },
```

Comment:
Maybe call this "hide"? I guess it's the reverse of "visible" from openui5

Reply:
Another thought, can they easily use browser API for this? (display: none, etc...)

Reply:
I prefer to keep the property and rename it to hide because the usage in frameworks will be smoother